### PR TITLE
chore(swaps): Removing swap native tokens experiment

### PR DESCRIPTION
**Pull request by dievazqu**
### Description

Removing swap native tokens experiment.

### Related issues
- Fixes RET-918

### CODE REVIEW

File: src/swap/SwapScreen.test.tsx [hunk 1]
```diff
@@ -278,7 +278,7 @@ describe('SwapScreen', () => {
     })
 
     mockExperimentParams.mockReturnValue({
-      swappingNonNativeTokensEnabled: false,
+      swapBuyAmountEnabled: true,
     })
```
Comment (jeanregisser) [inline src/swap/SwapScreen.test.tsx:282]: nit: could we even remove this?

**Review by jeanregisser — APPROVED** [review]
🚀

File: src/swap/SwapScreen.test.tsx (hunk 1 above)
Comment (dievazqu) [reply src/swap/SwapScreen.test.tsx:282]: Yes, there is another [task](https://linear.app/valora/issue/RET-920/remove-swap-buy-amount-experiment) to remove this. So we do it in separate PRs

**Review by bakoushin — APPROVED** [review]
Looks great! I was confused by the same thing as @jeanregisser

File: src/swap/SwapScreen.test.tsx (hunk 1 above)
Comment (jeanregisser) [reply src/swap/SwapScreen.test.tsx:282]: Ok I got confused as I didn't see that value in the existing test.

Comment (dievazqu) [reply src/swap/SwapScreen.test.tsx:282]: There was an error in a test if I removed the `mockExperimentParams.mockReturnValue` completely.
I'll handle that in the other task.

**Pull request merged**